cardano-node/src/Cardano/Tracing/ToObjectOrphans.hs

```diff
@@ -659,14 +659,14 @@ instance (Condense (HeaderHash blk), ProtocolLedgerView blk, SupportedBlock blk,
     TraceFoundIntersection _ _ _ ->
       mkObject [ "kind" .= String "ChainSyncClientEvent.TraceFoundIntersection" ]
 
-instance StandardHash blk => ToObject (TraceChainSyncServerEvent blk b) where
+instance (Show b, StandardHash blk) => ToObject (TraceChainSyncServerEvent blk b) where
     toObject _ ev = case ev of
-      TraceChainSyncServerRead tip _ ->
+      TraceChainSyncServerRead _ update ->
         mkObject [ "kind" .= String "ChainSyncServerEvent.TraceChainSyncServerRead"
-                 , "block" .= (String (pack . show $ tipPoint tip)) ]
-      TraceChainSyncServerReadBlocked tip _ ->
+                 , "block" .= (String (pack $ show update)) ]
```

Comment:
`update` has type `ChainUpdate blk b`:
```haskell
data ChainUpdate blk b
  = AddBlock b
  | RollBack (Point blk)
```
This type has a stock `Show` instance. As the type parameter `b` will be instantiated to `blk` or `Header blk`, this `show update` will print the whole block or header in case of the `AddBlock` constructor, which is a big no no :slightly_smiling_face:. Furthermore, isn't there a prettier way to print a `Point blk` that can be used for `RollBack`?

You'll need to write a custom `show`-like function for `ChainUpdate`. Maybe you can add a `Transformable` instance for it?